client/src/components/Home.js

```diff
@@ -1,43 +1,44 @@
-import { useEffect, useState } from 'react';
-import styled from 'styled-components';
+import { useEffect, useState } from "react";
+import styled from "styled-components";
```

Comment:
what code formatter are you using? single quotes are usually the standard in modern js.
We should all be using the same one or we'll keep having changes like this in our PRs. I use Prettier in vscode.

Reply:
I have prettier. I think with my settings it uses double quotes for imports and single quotes otherwise.

Reply:
Can we change that?

Reply:
Can we change that?  create-react-app boilerplate uses single quotes everywhere as well

Reply:
Yup, working on it.

Reply:
@aziz512 Fixed.